public/assets/inst.js

```diff
@@ -525,17 +528,20 @@ function runInstruction(raw) { //, RISCV) {
                 RISCV.gen_reg[((raw >>> 7) & 0x1F)] = new Long(RISCV.gen_reg[((raw >>> 7) & 0x1F)].getLowBitsUnsigned(), 0x155);
             }
             RISCV.pc += 4;
+            RISCV.instruction_amounts['arithmetic'] = RISCV.instruction_amounts['arithmetic'] + 1;
```

Comment:
what if you just write RISCV.instructionAmounts['arithmetic']++;

Reply:
did this